src/gerrit.py

```diff
@@ -40,7 +40,26 @@ def __call__(self, request: PreparedRequest):
         request.prepare_cookies(self.cookie_jar)
         return request
 
-class Gerrit(object):
+
+class FakeGerrit(object):
```

Comment:
nit: Perhaps `DryRunGerrit`

This would more symmetric with GerritGit below (assuming we recapitalize to `DryRunGerritGit`